src/content/self-hosted/install/backup-restore.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Backup and Restore
+description: Backup and Restore your Okteto instance
+id: backup
+sidebar_label: Backup and Restore
+---
+
+Backup and restore planning is essential to ensure the availability and integrity of your Okteto instance in case of unexpected failures or disaster recovery.
+This guide outlines the steps to recover an Okteto instance in such scenarios.
+
+## Critical Resources
+
+To effectively recover an Okteto instance, focus on the following critical resources:
+
+- **Service Accounts**: Preserve service accounts associated with users. These service accounts reside within the `okteto` namespace and are labeled with `dev.okteto.com: "true"`.
+- **Secrets**: Recreate all secrets within the `okteto` namespace.
+- **Custom Resources**: Restore Custom Resources (CRs) corresponding to Okteto's CRDs. They can be identified with the label `app.kubernetes.io/part-of: okteto`.
+
+## Recovery Process
+
+You can follow two different recovery processes:
+
+- **Light recovery processes**: recovers all the users’ metadata, tokens and secrets, but it doesn’t recover users’ namespaces and volumes. All this information is available on the `okteto` namespace.
```

Comment:
@pchico83 I'm wondering if it might be more clear to say something like:

- User only recovery
- User & data recovery

Something that's descriptive of what's being recovered. This seems like the general line of demarcation for what we recover in light vs. full, but WDYT?

Reply:
Right, much more clear. I will update